File: src/main/kotlin/inflearn/java/ch02/_2_2_보이는_학생.java
package inflearn.java.ch02;

import java.util.Scanner;
import java.util.StringTokenizer;

public class _2_2_보이는_학생 {
    public static int solution(int[] ints) {
        int prev = Integer.MIN_VALUE;

        int answer = 0;
        for (int cm : ints) {
            if(cm > prev) {
                answer++;
                prev = cm;
            }
        }
        return answer;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = Integer.parseInt(sc.nextLine());
        StringTokenizer st = new StringTokenizer(sc.nextLine());

        int[] ints = new int[n];
        for (int i = 0; i < n; i++) {
            ints[i] = Integer.parseInt(st.nextToken());
        }

        System.out.println(solution(ints));
    }
}

File: src/main/kotlin/inflearn/kotlin/ch02/2_2 보이는 학생.kt
package inflearn.kotlin.ch02

import java.util.*

object `2_2 보이는 학생` {
    fun solution(ints: IntArray): Int {
        var prev = Int.MIN_VALUE
        var answer = 0
        for (cm in ints) {
            if (cm > prev) {
                answer++
                prev = cm
            }
        }
        return answer
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        val n = sc.nextLine().toInt()
        val st = StringTokenizer(sc.nextLine())
        val ints = IntArray(n)
        for (i in 0 until n) {
            ints[i] = st.nextToken().toInt()
        }
        println(solution(ints))
    }
}